<launch>

    <node pkg="robotics_project_one" type="odom" name="odometryNode" output="screen" />

    <param name="r" type="double" value="0.07" />
    <param name="lx" type="double" value="0.2" />
    <param name="ly" type="double" value="0.169" />

</launch>